src/book/mod.rs

```diff
@@ -221,7 +221,14 @@ impl MDBook {
 
             debug!("[*]: Writing to .gitignore");
 
+            // Write the destination folder to file
             writeln!(f, "{}", relative).expect("Could not write to file.");
+            
+            // Add common files
+            let ignores = vec!("*~", ".DS_Store");
```

Comment:
I agree, temporary files are very specific to the environment of the user, we can not possibly support all environments and it should not be our responsibility to do so.